.github/workflows/test.yml

```diff
@@ -0,0 +1,75 @@
+# Use pre-built docker containers to run our unit tests on different python versions.
+
+name:  Run Test Suite
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  py35:
+    name: Python 3.5
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Pull Dependency Image
+        run: docker pull hpc4cmb/toast-deps-py35:latest
```

Comment:
is there another repo to build the `toast-deps` docker container?

Reply:
Yes, whenever a new tag is pushed to the cmbenv repo, the toast-deps docker images are built and pushed to docker hub:

https://github.com/hpc4cmb/cmbenv/blob/master/.github/workflows/deploy.yml

So to update the toast-deps image, you make any changes to the package files (updating versions, etc) in the cmbenv repo.  Then make a cmbenv tag and toast-deps will be rebuilt.

Reply:
The toast-deps images are based on the upstream official `python:3.x-slim-buster` images, with compiled dependencies pip packages installed to `/usr/local`.